addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -605,3 +615,12 @@ def _l10n_in_edi_ewaybill_get_by_consigner(self, company, document_type, documen
         return self._l10n_in_edi_ewaybill_connect_to_server(
             company, url_path="/iap/l10n_in_edi_ewaybill/1/getewaybillgeneratedbyconsigner", params=params
         )
+
+    @api.model
+    def _l10n_in_edi_ewaybill_get_error_code(self, company):
+        is_authenticated = self._l10n_in_edi_ewaybill_check_authentication(company)
+        if not is_authenticated:
+            return {}
+        return self._l10n_in_edi_ewaybill_connect_to_server(
+            company, url_path="/iap/l10n_in_edi_ewaybill/1/update_error_code", params={}
```

Comment:
so we agreed this is not needed (but we will keep the route on the iap server to have a quick check on our side if we need to do an update in the codes below)